modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/Checkbox.js

```diff
@@ -32,12 +29,10 @@ export default function Checkbox({
 
 	return (
 		<ClayCheckbox
+			aria-label={label}
```

Comment:
This should be done via Dynamic Attributes. Take a look to [ButtonTag](https://github.com/liferay-frontend/liferay-portal/blob/430053d3019fe2a6e2ef13a800e4800e17b1902b/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ButtonTag.java#L42)